modules/sdp/SDP.js

```diff
@@ -557,15 +557,15 @@ SDP.prototype.jingle2media = function(content) {
 
     const media = { media: desc.attr('media') };
 
-    media.port = '1';
+    media.port = '9';
     if (content.attr('senders') === 'rejected') {
         // estos hack to reject an m-line.
         media.port = '0';
     }
     if (transport.find('>fingerprint[xmlns="urn:xmpp:jingle:apps:dtls:0"]').length) {
-        media.proto = sctp.length ? 'DTLS/SCTP' : 'RTP/SAVPF';
+        media.proto = sctp.length ? 'DTLS/SCTP' : 'UDP/TLS/RTP/SAVPF';
```

Comment:
While you are editing this could you also update `DTLS/SCTP` into `UDP/DTLS/SCTP` please (which is also supported by all major browsers for quite some time now)?

Reply:
I was wondering about that but what is the state of sctp support even?

Reply:
You mean in Jitsi Meet? It is not being used on m.j.s, but people can still turn it own for their own deployments AFAIK. So that is why I think it's worth updating.

Reply:
agree, will do a followup change ;-)